user/nix/files/nix-daemon.dinit

```diff
@@ -0,0 +1,4 @@
+type = process
```

Comment:
assuming this continuously logs any realistic amount of stuff buffer is not a way to go

also should respect correct syntax for `=` vs `:`